src/wiki/plugins/editsection/wiki_plugin.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.conf.urls import url
+from wiki.core.plugins import registry
+from wiki.core.plugins.base import BasePlugin
+from wiki.plugins.editsection.markdown_extensions import EditSectionExtension
+
+from . import settings, views
+
+
+class EditSectionPlugin(BasePlugin):
+
+    slug = settings.SLUG
+    urlpatterns = {'article': [
+        url('^(?P<location>[0-9-]+)/header/(?P<header>\w+)/$',
+            views.EditSection.as_view(),
+            name='editsection'),
+    ]}
+
+    markdown_extensions = [EditSectionExtension()]
+
+    def __init__(self):
```

Comment:
This doesn't seemed to be needed. `BasePlugin` does not define an `__init__`-method.

Reply:
In that case, to be consistent, we should remove them from all plugins.
Every of the following plugins has currently an empty \_\_init__():

    attachments, editsection, globalhistory, help, images, links, macros, notifications

Only `haystack` has non, but that one is not based on `BasePlugin`. So
currently it's consistent with the \_\_init__()'s.

Any preference here?

Reply:
@floemker you are probably looking at an older version of the codebase... @atombrella has already been reading through most lines and removed such abnormalities :) Check out the current version of the `master` branch.

Reply:
Aah, right. These CTORs were already removed Oct 7 last year.
That answers as well my question. Removed the CTOR.